x-pack/plugins/maps/public/actions/layer_actions.ts

```diff
@@ -189,19 +189,19 @@ export function setLayerVisibility(layerId: string, makeVisible: boolean) {
       dispatch<any>(cleanTooltipStateForLayer(layerId));
     }
 
-    await dispatch({
```

Comment:
Removed await because dispatching a pure action is not asynchronous. Also cleaned up unneeded `async` declarations in `setLayerVisibility` and `toggleLayerVisible` since no awaits are used in these methods and no awaits are used by their callers